alpharaw/match/psm_match.py

```diff
@@ -30,6 +30,25 @@
 class PepSpecMatch:
     """
     Extract fragment ions from MS2 data.
+
+    Parameters
+    ----------
+    charged_frag_types : list, optional
+        fragment types with charge states,
+        e.g. ['b_z1', 'y_z2', 'b_modloss_z1', 'y_H2O_z2'].
+        If None, it is `get_charged_frag_types(['b','y','b_modloss','y_modloss'], 2)`.
+        By default None.
+
+    match_closest : bool, optional
+        if True, match the closest peak for a m/z;
+        if False, matched the higest peak for a m/z in the tolerance range.
+        By default True.
+
+    use_ppm : bool, optional
+        If use ppm, by default True.
+
+    tol_value : float, optional
+        tolerance value, by default 20.0
     """
 
     match_closest: bool = True
```

Comment:
I think this will be a class variable which will be shared across all instances. Is this intended?